pennylane/measurements/__init__.py

```diff
@@ -103,8 +103,10 @@ def circ(x, y):
 reset to the :math:`|0 \rangle` state by setting the ``reset`` keyword argument of ``qml.measure`` to ``True``.
 
 Users can also collect statistics on mid-circuit measurements along with other terminal measurements. Currently,
-``qml.expval``, ``qml.probs``, ``qml.sample``, ``qml.counts``, and ``qml.var`` are supported. Users have the
-ability to collect statistics on single measurement values.
+``qml.expval``, ``qml.probs``, ``qml.sample``, ``qml.counts``, and ``qml.var`` are supported. ``qml.probs``,
```

Comment:
This is written below the code example:

>PennyLane also supports postselecting on mid-circuit measurement outcomes. To learn more, refer to the documentation
of :func:`~.pennylane.measure`.

I would vote to not add details here, also because there is a plan to make an MCM quickstart page, which can go into depth on this topic and be linked from here, then.